### ISSUE TITLE
Question: Release version wrong/typo in git tag? 15.13.2 instead of 5.13.2?

### ISSUE THREAD

**Issue by mims-github** [sdkman/sdkman-cli#1051]
```ATTENTION: A new version of SDKMAN is available...

The current version is 15.13.2, but you have 5.13.1.

Would you like to upgrade now? (Y/n)n   
```
https://github.com/sdkman/sdkman-cli/releases/tag/15.13.2

@marc0der

PS: Thanks for this very helpful tool!

**Comment by DennisBecker** [comment]
Same issue, pls remove this wrong release.

**Comment by hgeraldino** [comment]
Thanks for reporting this @mims-github @DennisBecker 

This was released again with the correct version `5.13.2`

**Comment by marc0der** [comment]
Thanks for reporting this both, this happens when releasing before you've had your morning coffee! 😄

This issue has now been corrected.
